### PR TITLE
Update harness-governance-overview.md

### PR DESCRIPTION
**Pull request by thisrohangupta**
# Harness Developer Pull Request

Two new sections, authored by Rohan and reviewed by Doug, starting here: [Connectors](https://64b6ec25dc815203b4fc06c3--harness-developer.netlify.app/docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview/#connectors)

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md [hunk 1]
```diff
@@ -207,6 +207,28 @@ Policy Sets can be configured to evaluate policies on these Feature Flag events:
 
 See [Use Harness Policy As Code for Feature Flags](/docs/platform/Governance/Policy-as-code/using-harness-policy-engine-for-feature-flags).
 
+#### Connectors
+
+Policies are evaluated against Harness [Connectors](https://developer.harness.io/docs/category/connectors)
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md:212]:
```suggestion
Policies are evaluated against Harness [Connectors](https://developer.harness.io/docs/category/connectors).
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md [hunk 2]
```diff
@@ -207,6 +207,28 @@ Policy Sets can be configured to evaluate policies on these Feature Flag events:
...
+
+Policy Set can be configured to evaluate policies on the save of a connector.
+
+See [Use Harness Policy As Code for Connectors](https://developer.harness.io/docs/platform/Governance/Policy-as-code/sample-policy-use-case#connector-policy-samples)
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md:216]:
```suggestion
See [Use Harness Policy As Code for Connectors](https://developer.harness.io/docs/platform/Governance/Policy-as-code/sample-policy-use-case#connector-policy-samples).
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md [hunk 3]
```diff
@@ -207,6 +207,28 @@ Policy Sets can be configured to evaluate policies on these Feature Flag events:
...
+
+
+#### Template
+
+:::note
+Currently, this feature is behind the feature flag, `CDS_OPA_TEMPLATE_GOVERNANCE`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+
+You can configure policies on templates during the creation of the template. This ensures users follow the best practices set by their organization without the organization managing each template’s creation directly. By setting guard rails at creation time it reduces the number of checks needed at runtime to enforce standards.
```
Comment (douglas-j-bothwell) [inline docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md:224]:
```suggestion
You can configure policies on templates during the creation of the template. This ensures that users follow the best practices set by their organization without the organization managing each template’s creation directly. By setting guardrails at creation time, you can reduce the number of checks needed at runtime to enforce standards.
```

**Review by douglas-j-bothwell — APPROVED** [review]

File: docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md [hunk 4]
```diff
@@ -207,6 +207,29 @@ Policy Sets can be configured to evaluate policies on these Feature Flag events:
 
 See [Use Harness Policy As Code for Feature Flags](/docs/platform/Governance/Policy-as-code/using-harness-policy-engine-for-feature-flags).
 
+#### Connectors
+
+Policies are evaluated against Harness [Connectors](https://developer.harness.io/docs/category/connectors).
+
+Policy Set can be configured to evaluate policies on the save of a connector.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md:214]:
```suggestion
You can configure a Policy Set to evaluate policies when you save a container.
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md [hunk 5]
```diff
@@ -207,6 +207,29 @@ Policy Sets can be configured to evaluate policies on these Feature Flag events:
...
+
+See [Use Harness Policy As Code for Connectors](https://developer.harness.io/docs/platform/Governance/Policy-as-code/sample-policy-use-case#connector-policy-samples).
```
Comment (brian-f-harness) [inline docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md:216]:
```suggestion
Go to [Use Harness Policy As Code for Connectors](https://developer.harness.io/docs/platform/Governance/Policy-as-code/sample-policy-use-case#connector-policy-samples).
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md [hunk 6]
```diff
@@ -207,6 +207,29 @@ Policy Sets can be configured to evaluate policies on these Feature Flag events:
...
+
+
+#### Template
+
+:::note
+Currently, this feature is behind the feature flag, `CDS_OPA_TEMPLATE_GOVERNANCE`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+:::
+
+You can configure policies on templates during the creation of the template. This ensures that users follow the best practices set by their organization without the organization managing each template’s creation directly. By setting guardrails at creation time, you can reduce the number of checks needed at runtime to enforce standards.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md:225]:
```suggestion
You can configure template policies during the creation of the template. This ensures that you follow your organization's established best practices without the need to manage each template directly. By setting guardrails at creation time, you can reduce the number of checks required at runtime to enforce standards.
```

File: docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md [hunk 7]
```diff
@@ -207,6 +207,29 @@ Policy Sets can be configured to evaluate policies on these Feature Flag events:
...
+
+Users can now manage the configuration of Templates at configuration time by running policies on Template Save. During Pipeline Save, users can enforce that a specific template is being used. On Pipeline Run users can ensure that the resolved contents of the pipeline meet execution standards.
```
Comment (brian-f-harness) [inline docs/continuous-delivery/x-platform-cd-features/advanced/cd-governance/harness-governance-overview.md:227]:
```suggestion
You can now manage template configuration by running policies on Template Save. During Pipeline Save, you can enforce the use of a specific template. On Pipeline Run, you can ensure that the resolved contents of the pipeline meet execution standards.
```

**Review by brian-f-harness — APPROVED** [review]
lgtm. Added a few suggestions. Not sure if they apply.

**Pull request merged**